theme/templates/resource-landing-page/funding-agency.html

```diff
@@ -74,7 +74,7 @@ <h4 class="modal-title" id="add-funding-label"> ${mode} Funding Agency</h4></div
 
                                     <div class="controls">
                                         <input class="form-control input-sm urlinput" maxlength="200"
-                                            name="agency_url" type="url" v-model="currentlyEditing.agency_url" @keyup="checkAgencyName">
+                                            name="agency_url" type="url" v-model="currentlyEditing.agency_url" @keyup="checkAgencyName" readonly>
```

Comment:
nice check on this `readonly`
I agree that it seems like we wouldn't want the user to be able to edit the agency_url.
But we actually want to allow editing the url. This was discussed in initial implementation and decided best to allow the user additional flexibility